.github/workflows/tests.yml

```diff
@@ -41,7 +41,7 @@ jobs:
   mysql:
     strategy:
       matrix:
-        dbversion: ['mysql/mysql-server:latest', 'mysql:5.7']
+        dbversion: ['mysql:9.0', 'mysql:8.4', 'mysql:8.0', 'mysql:5.7']
```

Comment:
Can you change it to   
```
'mysql:9', 'mysql:8', 'mysql:5.7'
```

Reply:
@jinzhu 
Updated as requested, thank you!